afew/MailMover.py

```diff
@@ -72,31 +75,45 @@ def move(self, maildir, rules):
                     if self.dry_run:
                         continue
                     try:
-                        shutil.copy2(fname, self.get_new_name(fname, destination))
-                        to_delete_fnames.append(fname)
-                    except shutil.SameFileError:
-                        logging.warn("trying to move '{}' onto itself".format(fname))
-                        continue
+                        logging.info("Copy original mail: "+fname)
```

Comment:
please use `.format` instead of concatenating strings: https://pyformat.info/

Reply:
ok I will upgrade those